lib/modules/manager/composer/__fixtures__/composer1.json

```diff
@@ -6,6 +6,7 @@
   },
   "require": {
     "php": ">=5.3.2",
+    "ext-intl": "*",
```

Comment:
why we need this change?

Reply:
I think it was in order to have another "ignored" update because now `php` is not ignored.